cliquet/events.py

```diff
@@ -0,0 +1,15 @@
+from cliquet.utils import strip_uri_prefix
+
+
+class CollectionChanged(object):
+    """Triggered when a collection is changed.
+    """
+    def __init__(self, action, collection, request):
+        self.request = request
+
+        self.payload = {'collection_id': collection.record_id,
+                        'timestamp': collection.timestamp,
+                        'action': action,
+                        'uri': strip_uri_prefix(request.path),
+                        'user_id': request.unauthenticated_userid,
```

Comment:
I would suggest to use `request.prefixed_userid` that contains the auth policy name in addition to the user id